guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -7,94 +7,35 @@ You might want to change the encryption settings for {Project} depending on the
 [[Disabling_Weak_SSL_2-0_and_3-0_Encryption]]
 === Disabling Weak SSL 2.0 and SSL 3.0 Encryption
 
-If your {Project} fails Nessus scans because of SSL vulnerabilities, or your security infrastructure requires that you disable SSL 2.0 and SSL 3.0, you can edit the `/etc/foreman-installer/custom-hiera.yaml` file to remove weak encryption.
+Apache and Qpid services in {Project} use SSL 2.0 and SSL 3.0 encryption by default. If your {Project} or {SmartProxy} fails Nessus scans because of SSL vulnerabilities, or your security infrastructure requires that you disable SSL 2.0 and SSL 3.0, complete this procedure on {Project} or {SmartProxy} to remove weak encryption.
```

Comment:
@ekohl can you help to verify the first sentence here. Is that correct that Apache and Qpid use SSL 2.0 and 3.0?
You suggested changing the title, can you elaborate on how to change it exactly?

Reply:
I have a default installation of Satellite 6.7. I see the following on it: 
```
PORT    STATE SERVICE
443/tcp open  https
| ssl-enum-ciphers: 
|   SSLv3: No supported ciphers found
|   TLSv1.0: 
|     ciphers: 
|       TLS_DHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_RSA_WITH_AES_256_CBC_SHA - strong
|     compressors: 
|       NULL
|   TLSv1.1: 
|     ciphers: 
|       TLS_DHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_RSA_WITH_AES_256_CBC_SHA - strong
|     compressors: 
|       NULL
|   TLSv1.2: 
|     ciphers: 
|       TLS_DHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_DHE_RSA_WITH_AES_128_CBC_SHA256 - strong
|       TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 - strong
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA256 - strong
|       TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 - strong
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256 - strong
|       TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 - strong
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 - strong
|       TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 - strong
|       TLS_RSA_WITH_AES_128_CBC_SHA - strong
|       TLS_RSA_WITH_AES_128_CBC_SHA256 - strong
|       TLS_RSA_WITH_AES_128_GCM_SHA256 - strong
|       TLS_RSA_WITH_AES_256_CBC_SHA - strong
|       TLS_RSA_WITH_AES_256_CBC_SHA256 - strong
|       TLS_RSA_WITH_AES_256_GCM_SHA384 - strong
|     compressors: 
|       NULL
|_  least strength: strong
```
I see that qpid is running when entering `systemctl status qpidd`. Sounds like it does not use SSL by default as well.

Reply:
It would have surprised me if it did, especially SSL v2 which is really really ancient. Even SSLv3 was considered insecure back in 2014 which is already 6 years ago. If RHEL shipped something insecure by default, that would have surprised me.

Reply:
Shall we then dump this whole chapter 7. Disabling Weak Encryption because it is irrelevant is the current RHEL?

Reply:
Wait, 7.2. Disabling 64-bit Block Size Cipher Suites (SWEET32) is still relevant.

Reply:
> Shall we then dump this whole chapter 7. Disabling Weak Encryption because it is irrelevant is the current RHEL?

I'd say that currently the instructions are valid for some. Various institutions only allow TLS 1.2 (I think PCI requires it?). Since the default install allows TLS 1.0, that is still useful. However, it should be clearly stated that its disabling TLS 1.0 and 1.1, not SSL v2/v3.

Reply:
> Wait, 7.2. Disabling 64-bit Block Size Cipher Suites (SWEET32) is still relevant.

I think your example proves it doesn't. I don't see DES ciphers in there. Looks like it can be dropped.

Reply:
Are you sure that it can be dropped? Ganesh has a customer who used this in Satellite 6.5, see [BZ#1683832](https://bugzilla.redhat.com/show_bug.cgi?id=1683832).

Reply:
I can't speak for 6.5 since that's rather ancient. It should have the changes. Note that the BZ is about making the changes persistent. However, I think that the entire instructions are no longer needed.

Reply:
Used an internal reference Satellite 6.5 and that is not vulnerable to it. It was relevant on older releases but those are now EOL.

Reply:
I am more than happy to dump it then